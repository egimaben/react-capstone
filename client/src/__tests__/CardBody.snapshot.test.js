import React from 'react';
import renderer from 'react-test-renderer'
import CardBody from '../components/Card/CardBody';

console.error = err => { throw new Error(err); };
console.warn = warning => { throw new Error(warning); };

it('card renders correctly', () => {
  const rendered = renderer.create(<CardBody />);
  expect(rendered.toJSON()).toMatchSnapshot();
});