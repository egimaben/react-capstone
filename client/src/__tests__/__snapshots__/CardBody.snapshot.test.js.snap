// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card renders correctly 1`] = `
<div
  className="CardBody-cardBody-1"
/>
`;
